pypeit/core/coadd.py

```diff
@@ -2164,15 +2171,60 @@ def combspec(waves, fluxes, ivars, gpms, sn_smooth_npix,
     _fluxes = [np.float64(flux) for flux in fluxes]
     _ivars = [np.float64(ivar) for ivar in ivars]
 
+    # exposures that will not be coadded
```

Comment:
Change to a list. You could also call it a exp_gpm or something to make it more clear.